qiskit/synthesis/__init__.py

```diff
@@ -118,7 +118,7 @@
     synth_cnot_count_full_pmh,
     synth_cnot_depth_line_kms,
 )
-from .linear_phase import synth_cz_depth_line_mr, synth_cnot_phase_aam
+from .linear_phase import synth_cz_depth_line_mr, synth_cx_cz_line_my, synth_cnot_phase_aam
```

Comment:
In order to make this new function appear in the API documentation, please **also** add `synth_cx_cz_line_my` in line 49 (after `synth_cz_depth_line_mr`)

Reply:
Added--thank you for pointing it out!